WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController+Extensions.swift

```diff
@@ -241,6 +246,128 @@ extension NotificationsViewController: WPTableViewHandlerDelegate
 
 
 
+// MARK - Filter Helpers
+//
+extension NotificationsViewController
+{
+    func showFiltersSegmentedControlIfApplicable() {
+        guard tableHeaderView.alpha != WPAlphaZero && shouldDisplayFilters == true else {
```

Comment:
Should this not be `tableHeaderView.alpha == WPAlphaZero`? You only want to animate to full if it's currently zero? The original code read: `if (self.tableHeaderView.alpha == WPAlphaZero && self.shouldDisplayFilters) {`